tests/integration/test_cscmpy.py

```diff
@@ -0,0 +1,321 @@
+        ):
+            res_scen_2015_emms = res.filter(variable=variable, year=2015, scenario=scen)
+            if res_scen_2015_emms.empty:
+                raise AssertionError("No CO2 emissions data for {}".format(scen))
+
+            assert not np.equal(res_scen_2015_emms.values, 0).all()
+
+        ssp245_ghg_erf_2015 = res.filter(
+            variable="Effective Radiative Forcing|Greenhouse Gases",
+            year=2015,
+            scenario="ssp245",
+            run_id=1,
+        )
+        ssp245_ghg_erf_2014 = res.filter(
+            variable="Effective Radiative Forcing|Greenhouse Gases",
+            year=2014,
+            scenario="ssp245",
+            run_id=1,
+        )
+        # check that jump in GHG ERF isn't there
+        assert (
+            ssp245_ghg_erf_2015.values.squeeze() - ssp245_ghg_erf_2014.values.squeeze()
+        ) < 0.1
+        ssp245_ch4_conc_2015 = res.filter(
+            variable="Atmospheric Concentrations|CH4",
+            year=2015,
+            scenario="ssp245",
+            run_id=1,
+        )
+        ssp245_ch4_conc_2014 = res.filter(
+            variable="Atmospheric Concentrations|CH4",
+            year=2014,
+            scenario="ssp245",
+            run_id=1,
+        )
+        # ch
+        # check that jump in GHG ERF isn't there
+        assert (
+            ssp245_ch4_conc_2014.values.squeeze()
+            - ssp245_ch4_conc_2015.values.squeeze()
+        ) < 0.1
+
+        self._check_output(res, expected_output_file, update_expected_values)
+
+    @pytest.mark.ciceroscm
+    def test_variable_naming(self, test_scenarios):
+        missing_from_ciceroscm = (
+            "Effective Radiative Forcing|Aerosols|Direct Effect|BC|MAGICC AFOLU",
+            "Effective Radiative Forcing|Aerosols|Direct Effect|BC|MAGICC Fossil and Industrial",
+            "Effective Radiative Forcing|Aerosols|Direct Effect|OC|MAGICC AFOLU",
+            "Effective Radiative Forcing|Aerosols|Direct Effect|OC|MAGICC Fossil and Industrial",
+            "Effective Radiative Forcing|Aerosols|Direct Effect|SOx|MAGICC AFOLU",
+            "Effective Radiative Forcing|Aerosols|Direct Effect|SOx|MAGICC Fossil and Industrial",
+            "Heat Uptake|Ocean",
+            "Net Atmosphere to Land Flux|CO2",
+            "Net Atmosphere to Ocean Flux|CO2",
+        )
+        common_variables = [
+            c for c in self._common_variables if c not in missing_from_ciceroscm
+        ]
+        res = run(
+            climate_models_cfgs={
+                "CICEROSCMPY": [{"pamset_udm": {"lambda": 0.540}, "pamset_emiconc": {}}]
+            },
+            scenarios=test_scenarios.filter(scenario="ssp126"),
+            output_variables=common_variables,
+        )
+
+        missing_vars = set(common_variables) - set(res["variable"])
+        if missing_vars:
+            raise AssertionError(missing_vars)
+
+    @pytest.mark.ciceroscm
+    def test_w_out_config(self, test_scenarios):
```

Comment:
So if you're missing any of the config, it doesn't run?

Reply:
Not sure I quite understand the question, this method is mainly checking what happens if you send the out_config parameter set to anything but None. If you do, it should throw an error. However, there are some required configs which I should maybe also be testing for...

Reply:
Ah ok got it